src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -145,6 +151,9 @@ export class BeneficiariesService {
 				dataObject['enrollment_number'] =
 					data?.program_beneficiaries[0]?.enrollment_number;
 				records.push(dataObject);
+				dataObject['aadhar_no']=data?.aadhar_no; 
+				dataObject['aadhar_verified']=data?.aadhar_verified ? data?.aadhar_verified:'no';
```

Comment:
so do we get `yes / no / in_progress` here?

Reply:
yes sir